packages/x-date-pickers-pro/src/internals/hooks/useMultiInputRangeField/useMultiInputDateTimeRangeField.ts

```diff
@@ -82,6 +82,8 @@ export const useMultiInputDateTimeRangeField = <TDate, TTextFieldSlotProps exten
     onChange,
     disabled,
     readOnly,
+    selectedSections,
```

Comment:
Note for anyone wondering, this file is changed, because these https://github.com/mui/mui-x/pull/10915 changes were only merged to master and didn't end up in `next` during the transition.